### PR TITLE
show assignee on assigned task

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
Once a task is assigned we need to show the assignee name

### CODE REVIEW

File: src/components/issues/ActionForm.tsx [hunk 1]
```diff
@@ -43,12 +44,26 @@ const reducer = (state: ActionFormReducer, action: reducerAction) => {
 
 const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
     const [state, dispatch] = useReducer(reducer, initialState);
+    const [isAssigned, setIsAssigned] = useState(false);
     const [updateTask] = useUpdateTaskMutation();
+    const { data } = useGetTaskDetailsQuery(taskId);
+
+    useEffect(() => {
```
Comment (vvaibhavdesai) [inline src/components/issues/ActionForm.tsx:51]: why do you need an useEffect for this ?

File: src/components/issues/ActionForm.tsx [hunk 2]
```diff
@@ -63,39 +78,47 @@ const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
...
+                    <select
+                        name="status"
+                        id="status"
+                        value={state.status}
+                        onChange={(e) =>
+                            dispatch({ type: 'status', value: e.target.value })
+                        }
+                        className={styles.assign}
+                    >
+                        {taskStatus.map(([name, status]) => (
```
Comment (vvaibhavdesai) [inline src/components/issues/ActionForm.tsx:114]: are we sure about taskStatus not being undefined or null?

Comment (RitikJaiswal75) [reply src/components/issues/ActionForm.tsx:114]: yes

File: src/components/issues/ActionForm.tsx (hunk 1 above)
Comment (RitikJaiswal75) [reply src/components/issues/ActionForm.tsx:51]: it was giving the multiple re rendering error

**Review by vvaibhavdesai — APPROVED** [review]

**Review by DashDeipayan — APPROVED** [review]
LGTM

**Pull request merged**